website/src/docs/vue.md

```diff
@@ -35,7 +35,7 @@ Instead of adding a UI plugin to an Uppy instance with `.use()`, the Uppy instan
 ```html
 <template>
   <div id="app">
-    <dashboard :uppy="uppy" :plugins="['Webcam']"/>
+    <dashboard :uppy="uppy" :plugins="['Webcam']" :props="{theme: 'light'}" />
```

Comment:
`uppy` and `plugins` are already props, so now you have `props` inside props. I don't think that is something we want to document. Could you elaborate?

Reply:
The idea is to show how to setup the dashboard options (theme, metaFields, etc) 

I tried using this:

` <dashboard :uppy="uppy" :plugins="['Webcam']" :theme="light" />`

and also

` <dashboard :uppy="uppy" :plugins="['Webcam']" theme="light" />`

But they don't work. The example in my PR works.

Reply:
Maybe there's another way to do this, but I don't know how to do it :)

Reply:
You're right! I misinterpreted, as can be seen in the props docs section of [`<dashboard />`](https://uppy.io/docs/vue/#lt-dashboard-gt).